src/analyze.cpp

```diff
@@ -3371,74 +3371,64 @@ bool handle_is_ptr(TypeTableEntry *type_entry) {
 }
 
 void find_libc_include_path(CodeGen *g) {
+#ifdef ZIG_OS_WINDOWS
     if (!g->libc_include_dir || buf_len(g->libc_include_dir) == 0) {
+        Win32SDK win_sdk;
```

Comment:
This should go on the `CodeGen` struct, and we should use the same value for `find_libc_include_path` and `find_libc_lib_path`.